SdkVersion.props

```diff
@@ -3,12 +3,13 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <SDKVersionRS2>10.0.15063.0</SDKVersionRS2>
-	<SDKVersionRS3>10.0.16299.0</SDKVersionRS3>
-	<SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
-	<SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
-	<SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
-	<SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
+	  <SDKVersionRS3>10.0.16299.0</SDKVersionRS3>
+	  <SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
+	  <SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
+	  <SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
+	  <SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
   </PropertyGroup>
+  
```

Comment:
there is no real change in this file

Reply:
CodeFlow by default doesn't show the whitespace changes (at least for me). I fixed the indents (which also used tabs before):

![image](https://user-images.githubusercontent.com/35987549/131558171-460fe84a-dc89-49e7-bd1e-17d2400b01fa.png)